### PR TITLE
mat icons

### PR DESCRIPTION
**Pull request by iamtarun1993**

### CODE REVIEW

File: src/app/fyle/help/support-dialog/support-dialog.page.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <ion-toolbar>
     <ion-buttons slot="end">
```
Comment (madhavmansuriya40) [inline src/app/fyle/help/support-dialog/support-dialog.page.html:3]: should consider using mode="md"

**Review by Julias0 — APPROVED** [review]

File: src/app/fyle/my-view-trips/advance-requests/advance-requests.component.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     </ion-title>
     <ion-buttons slot="end">
```
Comment (madhavmansuriya40) [inline src/app/fyle/my-view-trips/advance-requests/advance-requests.component.html:6]: mode="md"

File: src/app/fyle/my-view-trips/hotel-requests/hotel-requests.component.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     </ion-title>
     <ion-buttons slot="end">
```
Comment (madhavmansuriya40) [inline src/app/fyle/my-view-trips/hotel-requests/hotel-requests.component.html:6]: mode="md"

File: src/app/fyle/my-view-trips/advance-requests/advance-requests.component.html (hunk 2 above)
Comment (madhavmansuriya40) [reply src/app/fyle/my-view-trips/advance-requests/advance-requests.component.html:6]: wherever required

File: src/app/fyle/dashboard/enterprise-dashboard-card/enterprise-dashboard-card.component.html [hunk 4]
```diff
@@ -45,10 +42,10 @@
               <p class="dashboard-card--expanded-card-title m-0"> {{ stat.title }} </p>
             </div>
             <div *ngIf="this.item.title === 'trips'">
-              <ion-icon class="dashboard-card--icon-curve" *ngIf="stat && stat.warning && stat.count && stat.count > 0" class="dashboard-card--icon-card" src="../../../assets/svg/warning-inverted.svg" ></ion-icon>
+              <mat-icon class="dashboard-card--icon-card" *ngIf="stat && stat.warning && stat.count && stat.count > 0"  svgIcon="warning-inverted" ></mat-icon>
```
Comment (madhavmansuriya40) [inline src/app/fyle/dashboard/enterprise-dashboard-card/enterprise-dashboard-card.component.html:45]: `stat?.warning && stat?.count? > 0`

can we use this?

Comment (madhavmansuriya40) [reply src/app/fyle/dashboard/enterprise-dashboard-card/enterprise-dashboard-card.component.html:45]: if yes, can be replaced wherever needed

File: src/app/fyle/help/support-dialog/support-dialog.page.html (hunk 1 above)
Comment (iamtarun1993) [reply src/app/fyle/help/support-dialog/support-dialog.page.html:3]: I did nothing here, your code sir..

**Pull request closed without merging**